providers/ns1/ns1Provider.go

```diff
@@ -198,11 +199,44 @@ func (n *nsone) GetDomainCorrections(dc *models.DomainConfig) ([]*models.Correct
 				})
 			}
 		}
-		return corrections, nil
+
 	}
 
-	// Insert Future diff2 version here.
+	if diff2.EnableDiff2 {
```

Comment:
Please add the `	return corrections, nil` to the old diff section, then you can remove this line and the ending `}`.  The result will be the "git blame"-friendly version I mentioned earlier (in case it wasn't clear).

Reply:
heh, was just pushing exactly this :)